### PR TITLE
bug fix in common/sgx/collateral.c:_get_crl_validity

### PR DESCRIPTION
**Pull request by RRathna**
Signed-off-by: Rathna Ramesh <rathna1993@gmail.com>

### CODE REVIEW

**Comment by RRathna** [comment]
bors try

**Review by radhikaj — APPROVED** [review]

File: common/sgx/collateral.c [hunk 1]
```diff
@@ -91,7 +91,7 @@ static oe_result_t _get_crl_validity(
         {
             OE_CHECK_MSG(
                 oe_crl_get_update_dates(
-                    &crls[0], &crl_this_update_date, &crl_next_update_date),
+                    &crls[i], &crl_this_update_date, &crl_next_update_date),
```
Comment (mingweishih) [inline common/sgx/collateral.c:94]: It would be good if we can add a test to cover this behavior.

**Review by mingweishih — APPROVED** [review]

**Comment by RRathna** [comment]
bors try

**Comment by radhikaj** [comment]
bors try-

**Comment by rs--** [comment]
bors retry

**Comment by rs--** [comment]
bors try

**Pull request merged**